### DISCUSSION TITLE
Is sequelize lack dataType of 'SET'?

### DISCUSSION THREAD

**Discussion by older-fox** [sequelize/sequelize#15103] [Help & Questions] [answered]
As the title said, I can't find this type in the docement,
I know that I can extend it by use ```DataTypes.ABSTRACT```
But question is ```SET``` is a type that requires some values, and there's no any things had mentioned how to 'receive' those values.
Hope you can give me some help and thanks for your patient

**Comment by ephys** [comment, marked as answer]
You can pass values to your custom SET datatype through its constructor, just like we do for enum:

https://github.com/sequelize/sequelize/blob/main/src/data-types.js#L817

Feel free to open a feature request for a SET DataType though

**Reply by WikiRik** [reply]
We have https://github.com/sequelize/sequelize/issues/13826 already so if more dialects support it, we can just update that issue

**Reply by older-fox** [reply]
> You can pass values to your custom SET datatype through its constructor, just like we do for enum:
> 
> https://github.com/sequelize/sequelize/blob/main/src/data-types.js#L817
> 
> Feel free to open a feature request for a SET DataType though

Sure, thanks for help.
, Just curious why  don't you add it by default cuz the way to use SET just like ENUM,
